client/src/components/App/App.tsx

```diff
@@ -52,63 +33,64 @@ export default class App extends Component<{}, State> {
       .firestore()
       .collection('items')
       .onSnapshot(querySnapshot => {
-        let features: Feature[] = []
+        let items: FirebaseItem[] = [];
 
         querySnapshot.docs.forEach(doc => {
           const id = doc.id;
-
-          const { title, geo, fullPath, type } = doc.data();
-
-          const feature = {
-            type: 'Feature',
-            properties: {
-              id,
-              title,
-              fullPath,
-              type,
-            },
-            geometry: {
-              type: 'Point',
-              coordinates: [geo.longitude, geo.latitude],
-            },
-          }
-          features.push(feature)
+          const item = doc.data() as FirebaseItem;
+          items.push({
+            id,
+            ...item,
+          });
         });
 
         this.setState({
-          geodata: {
-            type: this.state.geodata.type,
-            features 
-          }
-        })
-      })
+          items,
+        });
+      });
   };
 
   render() {
-    if (!this.state.geodata.features.length) {
+    if (!this.state.items.length) {
       return <div className="App"></div>;
     }
 
     return (
       <div className="App">
         <Switch>
           <Route path="/" exact>
-            <Map geodata={this.state.geodata} />
+            <Map items={this.state.items} />
```

Comment:
macht hier vllt ein redirect mehr Sinn auf Map?

Reply:
Hm, aber wir haben zu dem Zeitpunkt noch keine Koordiaten um `/map/:lat/:lng/:zoom` auszufüllen? In der Map Component werden ja irgendwelche Defaults genutzt falls die Props nicht mitkommen. Oder meinst du dieser Redirect könnte die Defaults hier setzen?

Also literally:

```jsx
<Route path="/" exact>
  {# (lat,lng) for e.g. Berlin #}
  <Redirect to="/map/1234/1234/10" />
</Route>
```

Oder meinst du das anders?

Reply:
beim initialen Besuch wirds ja aktuell auch irgendwie gesetzt?

Wäre aber cool da einen redirect mit konfigurierbaren Variablen zu machen (für später). Aktuell wäre ein Zoom auf Europa auf jeden Fall super!

> Also literally:

:+1: